.travis.yml

```diff
@@ -10,35 +10,20 @@ cache:
 
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.3
```

Comment:
Yeah 1.9.3 is not supported but still in use so at least I prefer not to break it if I don't have to.